### ISSUE TITLE
How to write QA tests

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#3588]

**Comment by Obadha2** [comment]
@rowasc @Angamanga 
This needs better context, or clearer wording.
What is meant by QA tests?
   - Are these the tests that products go through - test cases? 
  -  Are these the testing procedure for products - test steps/procedures?
  - Are these the scripts that guide the testing process - test scripts?

**Comment by rowasc** [comment]
This is about test cases for the product yes...let me rephrase it.
How do you write a new manual test for platform?
Where do you document them?
(like testpad, checklists in pull requests, etc)"
thanks
